plugins/easings/sigmoid/README.md

```
@@ -0,0 +1,74 @@
+[![banner](https://particles.js.org/images/banner3.png)](https://particles.js.org)
+
+# tsParticles Easing Sigmoid Plugin
+
+[![jsDelivr](https://data.jsdelivr.com/v1/package/npm/@tsparticles/plugin-easing-sigmoid/badge)](https://www.jsdelivr.com/package/npm/@tsparticles/plugin-easing-sigmoid)
+[![npmjs](https://badge.fury.io/js/@tsparticles/plugin-easing-sigmoid.svg)](https://www.npmjs.com/package/@tsparticles/plugin-easing-sigmoid)
+[![npmjs](https://img.shields.io/npm/dt/@tsparticles/plugin-easing-sigmoid)](https://www.npmjs.com/package/@tsparticles/plugin-easing-sigmoid) [![GitHub Sponsors](https://img.shields.io/github/sponsors/matteobruni)](https://github.com/sponsors/matteobruni)
+
+[tsParticles](https://github.com/tsparticles/tsparticles) plugin for adding the easing sigmoid support.
+
+## How to use it
+
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)